spec/factories/tenants.rb

```diff
@@ -2,7 +2,7 @@
   factory :tenant, aliases: [:openstack_tenant], class: 'Atmosphere::Tenant' do |f|
     tenant_type 'private'
     tenant_id { SecureRandom.hex(4) }
-    site_id { 'somesite' }
+    site_id { "site_id_#{SecureRandom.hex(4)}" }
```

Comment:
I did this on purpose to "simulate" PLGrid-like behavior. Future tests will need to be careful to override this to produce meaningful results (the random assignment is more "VPH-like").